app/javascript/src/dashboard.js

```diff
@@ -116,24 +71,141 @@ $('document').ready(() => {
     }
   )
 
+  const handleAjaxError = e => {
+    if (e.status === 401) {
+      location.reload();
+    } else {
+      console.log(e);
+      if (e.responseJSON && e.responseJSON.error) {
+        alert(e.responseJSON.error);
+      } else if (e.response.statusText) {
+        alert(`Unable to complete your request: ${e.response.statusText}`);
+      } else {
+        alert("Unable to complete your request.");
+      }
+    }
+  };
+
   // Enable all data tables on dashboard but only filter on volunteers table
+  const editSupervisorPath = id => `/supervisors/${id}/edit`;
+  const editVolunteerPath = id => `/volunteers/${id}/edit`;
+  const casaCasePath = id => `/casa_cases/${id}`;
   var volunteersTable = $('table#volunteers').DataTable({
     autoWidth: false,
     stateSave: false,
-    columnDefs: [
+    columns: [
```

Comment:
Is there any particular reason this changed? Is this a datatables version thing?

Reply:
@compwron I honestly don't 100% remember why I made the change. _But_, `columnDefs` requires a `targets` property indicating the column's position in the table. `columns`, on the other hand, just uses the entered order, and each column requires its own configuration anyway.